server/api/externalapi.ts

```diff
@@ -97,6 +90,54 @@ class ExternalAPI {
 
     return `${this.baseUrl}${endpoint}${JSON.stringify(params)}`;
   }
+
+  private createAxiosInstance(
+    baseUrl: string,
+    params: Record<string, unknown>,
+    options: ExternalAPIOptions
+  ) {
+    const proxySettings = getSettings().main.proxy;
+    const privateIPAddressRegexp = new RegExp(
+      /(^192\.168\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])$)|(^172\.([1][6-9]|[2][0-9]|[3][0-1])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])$)|(^10\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])\.([0-9]|[0-9][0-9]|[0-2][0-5][0-5])$)/
```

Comment:
This regex doesn't work.  For example, `192.168.249.1` doesn't match.  Also, what about IPv6?